othello/main.py

```diff
@@ -210,31 +260,37 @@ def hardcore_training(black, white, board_size, total_iterations: int = 100_000,
 		black=CNNDQNTrainableAgent(
 			Color.BLACK,
 			immediate_reward=MinimaxHeuristic(global_config.board_size),
-			board_size=global_config.board_size
+			board_size=global_config.board_size,
+			start_epsilon=0.99,
+			end_epsilon=0.01,
+			epsilon_steps=20_000,
+			allow_exploration=True
 		),
 		train_black=True,
 		white=RandomAgent(Color.WHITE),
 		train_white=False,
-		num_episodes=1,
-		plot_win_ratio=False,
-		plot_win_ratio_live=False,
-		verbose=True,
-		verbose_live=True,
+		num_episodes=150,
+		plot_win_ratio=True,
+		plot_win_ratio_live=True,
+		verbose=False,
+		verbose_live=False,
 		random_start=True,
+		train_all_agents=True,
 	)
 	main()
```

Comment:
If you would make a list of configs here and then simply do:
```python
for config in configs:
    main()
```
All these changes in **main.py** and **config.py** become unnecessary.

Reply:
ok but I did not make the list!